src/backend/index.js

```diff
@@ -587,7 +591,7 @@ function processObservables (instance) {
 function scrollIntoView (instance) {
   const rect = getInstanceRect(instance)
   if (rect) {
-    window.scrollBy(0, rect.top)
+    window.scrollBy(0, rect.top + (rect.height - window.innerHeight) / 2)
```

Comment:
making it appear in the middle of the screen is pretty neat actually, It did bothered me some times (toolbars haha) when it scrolls to the very top

Reply:
> Not sure how to change this though (and I feel like it's a new change?)

What do you mean? 😕

Reply:
We have a component that's often taller then the screen, so when it scrolls to the middle, it's a bit odd. It would be good to scroll to middle for smaller components, but scroll to top for components taller than the screen height.

Reply:
Or making sure we never scroll more than half of the screen height